aws/src/test/java/org/apache/iceberg/aws/CatalogTest.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.aws;
+
+import org.apache.iceberg.CatalogType;
+import org.apache.iceberg.aws.dynamodb.DynamoDbCatalog;
+import org.apache.iceberg.aws.glue.GlueCatalog;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+public class CatalogTest {
+
+  @Test
+  public void testGlueCatalogTypeImpl() {
+    Assertions.assertThat(
+        CatalogType.getCatalogImpl(CatalogType.GLUE.getTypeName()))
+        .isEqualTo(GlueCatalog.class.getName());
+  }
+
+  @Test
+  public void testDynamoCatalogTypeImpl() {
+    Assertions.assertThat(
```

Comment:
Assertions should have context so it is clear what happened from the error message when they fail.